frontend/static/templates/home.html

```diff
@@ -0,0 +1,35 @@
+<html lang="en">
```

Comment:
File above ^ is `webpack-bundle.js` - no need to commit it. Given specific req from `yarn.lock` and your source it will get reproduced